File: solutions/beecrowd/1005/1005.php
<?php
$a = (float)trim(fgets(STDIN));
$b = (float)trim(fgets(STDIN));
$average = ($a * 3.5 + $b * 7.5) / 11.0;

echo "MEDIA = " . number_format($average, 5, '.', '') . "\n";
?>

File: solutions/beecrowd/1005/1005.exs
a = IO.gets("")
|> String.trim()
|> String.to_float()

b = IO.gets("")
|> String.trim()
|> String.to_float()

media = (a * 3.5 + b * 7.5) / 11.0
:io.format("MEDIA = ~.5f~n", [media])
